### DISCUSSION TITLE
From one pipeline to multiple pipelines

### DISCUSSION THREAD

**Discussion by espinielli** [ropensci/targets#943] [Help]
I have developed a pipeline that from a data file in `data/<country>_<from>_<until>.csv` generates
* various top N horizontal bar plots
* a Quarto report/webpage for `<country>` containing the plots above.

My Quarto report is `report.qmd` and the result of the pipeline is

* `report.html`
* `report_files/`

Ideally I would like it to be

`output/<country>/index.html`
`output/<country>/report_files/`

Then I would like to generalize this to multiple files/countries, i.e. I would like to apply my pipeline to all files of the form `<country>_<from>_<until>.csv` in order to have

`output/<country1>/index.html`
`output/<country1>/report_files/`
`output/<country2>/index.html`
`output/<country2>/report_files/`
...

Has anybody tried something similar?
Any suggestions as super welcome

**Comment by wlandau** [comment]
You could statically branch over `tar_quarto()` calls using `tarchtypes::tar_eval()`: https://books.ropensci.org/targets/static.html#metaprogramming. Either that or you could treat all those individual Quarto report files as a single Quarto project.
